unitary/examples/quantum_rpg/main_loop.py

```diff
@@ -40,6 +41,10 @@ def __init__(
         self.file = file
         self.party = party
 
+
+    def print_title_screen(self):
```

Comment:
Should this be called from `__init__` or is the caller supposed to explicitly print the title?

Reply:
I'd like to make it explicit, so that we can control the flow better.